### ISSUE TITLE
Forestry Bio Generators not usable

### ISSUE THREAD

**Issue by negative-seven** [xJon/Tekkit-2#66]
(posted by rythin-sr, but on wrong account)

**Basic details**
- **Are you using a legitimate Minecraft account?**: yes
- **Are you using Java 8, 64-bit?**: yes
- **Did you add any additional mods**: no
- **Crash/latest log link (if applicable)**:not applicable

**Describe the issue**

Forestry Bio Generators seem to produce EU with a bugged voltage, blowing up any machine it's connected to, including EV-Transformers.

**Comment by xJon** [comment]
Can you reproduce this issue with only Forestry and Industrial Craft Classic installed?
If so, can you also with IC2 Experimental?

**Comment by xJon** [comment]
You might be able to use an Adjustable Transformer as a workaround (pointed out by @DatGurney).

**Comment by Speiger** [comment]
Yeah this is a forestry issue. And unless they patch it this isn't possible.

**Comment by xJon** [comment]
Thanks for the heads up @Speiger. Closing this for now as Forestry is not in active development.

**Comment by xJon** [comment]
It seems simply using the Liquid Fuel Generator from IC2 Classic instead of Forestry's Bio Generator could be an acceptable solution.

**Comment by xJon** [comment]
Will be patched in the next update thanks to @ACGaming!
